### ISSUE TITLE
Resulting Setup.msi exits without any warning

### ISSUE THREAD

**Issue by d9k** [electron-userland/electron-builder#249]
I'm packing my electron app with electron-builder on Windows 7 x64. But when I try to execute resulting `dist\win-x64\Setup.msi` the installation process just exits without any message. How can I debug what goes wrong?
I tried to make my application "Squirrel Aware" just by copying code from http://www.mylifeforthecode.com/creating-a-windows-distribution-of-an-electron-app-using-squirrel/ but no luck (maybe it's outdated).
Also when I try another way and run `dist\win-x64\MyApp-0.1.0.exe` the application starts but I can't find whether application files were copied anywhere. And application state is not saved too. Application state and settings are intended to be stored at the `settings.json` which my application tries to create after settings changed or before exit. But when I run packaged `MyApp-0.1.0.exe` it "installs" again and no state is saved.

Am I asking the question at the wrong place? Should I create issue at `electron-packager` repo instead?
I have zero experience with Squirrel installers and lack of documentation exhausts me. Also packaging process is too long for understanding through experiments.

**Comment by develar** [comment]
> Should I create issue at electron-packager repo instead?

Such questions should be addressed to https://github.com/electronjs/windows-installer

**Comment by develar** [comment]
> Also packaging process is too long for understanding through experiments.

Consider to:
* build only one arch (`--arch x64` (or `ia32`))
* if you don't need an MSI, set `build.win.noMsi` to `true`. See https://github.com/loopline-systems/electron-builder#distributable-format-configuration

**Comment by d9k** [comment]
@develar After I run builder with `--arch x64` the `dist/win-x64` folder is empty. Is it normal behaviour?

**Comment by d9k** [comment]
@develar also I get dll linking error (as it was before you made extraResources support https://github.com/loopline-systems/electron-builder/issues/230) when I run `dist/win/Setup.exe` which was made by `builder --arch x64`

**Comment by d9k** [comment]
@develar my `package.json`:

```
{
  "license": "proprietary",
  "private": true,
  "build": {
    "dir": "app",
	"win": {
		"extraResources": [
			"./*.dll"
		]
	}
  },
  "scripts": {
    "postinstall": "install-app-deps",
    "start": "./node_modules/.bin/electron app",
    "dist": "./node_modules/.bin/build",
    "dist:win": "./node_modules/.bin/build --platform win32",
	"dist:x64": "./node_modules/.bin/build --arch x64"
  },
  "devDependencies": {
    "babel-cli": "^6.6.4",
    "babel-core": "^6.6.4",
    "babel-preset-es2015": "^6.6.0",
    "babel-preset-react": "^6.5.0",
    "babel-preset-stage-0": "^6.5.0",
    "babelify": "^7.2.0",
    "browserify": "^12.0.1",
    "electron-builder": "^2.10.1",
    "electron-prebuilt": "^0.36.0",
    "gulp": "^3.9.1",
    "gulp-babel": "^6.1.2",
    "gulp-concat": "^2.6.0",
    "gulp-notify": "^2.2.0",
    "gulp-sourcemaps": "^1.6.0",
    "gulp-util": "^3.0.7",
    "vinyl-source-stream": "^1.1.0",
    "watchify": "^3.6.1",
    "winreg": "0.0.13"
  }
}
```

When I run `npm run dist:win` it glitches too: no packages are made, only unpacked application files at `dist/OrbiletClient-win32-ia32`, `dist/OrbiletClient-win32-x64`.

**Comment by develar** [comment]
I filed https://github.com/electronjs/windows-installer/issues/40

**Comment by d9k** [comment]
@develar thanks, what about issues with `--arch x64` and `--platform win32` arguments?

**Comment by develar** [comment]
@d9k Is your last question fixed in the 3.0.1?

**Comment by d9k** [comment]
24.03.2016, 19:03, "Vladimir Krivosheev" <notifications@github.com>:@d9k Is your last question fixed in the 3.0.1?—You are receiving this because you were mentioned.Reply to this email directly or view it on GitHub  You mean different build options? I'll try later.___________________________________________С уважением,разработчик ООО "ОрбитальСофт" (orbilet.ru)Комаров Дмитрий Алексеевичskype:dmkom60

**Comment by d9k** [comment]
moved problems with arguments to separate issue https://github.com/electron-userland/electron-builder/issues/267
